### PR TITLE
Add load options

### PR DESCRIPTION
**Pull request by mzogheib**
Reference: #65 

### Usage
```
  const content = snippet.min({
    apiKey,
    load: {
      integrations: {
        All: false,
        'Google Analytics': true,
        AdRoll: true,
        Hotjar: true,
      },
    },
  });
```

* I don't believe the snippet tests need to be updated but please advise if I'm mistaken
* I npm-linked these changes to my own project and all working as expected
image

### CODE REVIEW

File: test/render.test.js [hunk 1]
```diff
@@ -101,6 +107,12 @@ describe('snippet', function() {
         'analytics.load("key")');
     });
 
+    it('should set the load options', function() {
+      assertContains(
+        snippet.max({ apiKey: 'key', load: { integrations: { All: false } } }),
+        'analytics.load("key", {"integrations":{"All":false}})');
+    });
+
```
Comment (juliofarah) [inline test/render.test.js:115]: seems like this test is a dupe from the one a few lines above?

**Comment by juliofarah** [comment]
@mzogheib Thank you so much for your contribution and I'm sorry we didn't have time to take a look at it earlier!
The changes look good to me, just left a small comment!

> I npm-linked these changes to my own project and all working as expected
Would you mind adding a screenshot showing the `analytics.Integrations` object, or any other screenshot showing the changes working end-to-end? I'll merge these and ship a new version as soon as you get back to me!

Cheers

File: lib/index.js [hunk 2]
```diff
@@ -87,5 +87,11 @@ function renderPage(page) {
 function renderLoad(settings) {
   if (!settings.load) return '';
 
+  if (typeof settings.load !== 'boolean') {
+    // eslint-disable-next-line no-restricted-globals
+    var loadOptions = JSON.stringify(settings.load);
+    return 'analytics.load("' + settings.apiKey + '", ' + loadOptions + ');';
+  }
```
Comment (juliofarah) [inline lib/index.js:94]: In order to be fully compliant with the typescript definitions added on #55 it'd be great if we could change the type of `load?: boolean` to something like 

```
type LoadOptions = boolean | { ... }

// ...
load?: LoadOptions
```

**Comment by mzogheib** [comment]
> @mzogheib Thank you so much for your contribution and I'm sorry we didn't have time to take a look at it earlier!
> The changes look good to me, just left a small comment!
> 
> > I npm-linked these changes to my own project and all working as expected
> 
> Would you mind adding a screenshot showing the `analytics.Integrations` object, or any other screenshot showing the changes working end-to-end? I'll merge these and ship a new version as soon as you get back to me!
> 
> Cheers

Thanks @juliofarah and no problem! I'll provide this soon and address your other comments too.

File: test/render.test.js (hunk 1 above)
Comment (mzogheib) [reply test/render.test.js:115]: Yep, the first is for the `describe('#max', ... )` block and the second for the `describe('#min', ...)`. I noticed all other tests were done this way so just followed that same pattern. Happy to do it a different way if you have a recommendation.

File: lib/index.js (hunk 2 above)
Comment (mzogheib) [reply lib/index.js:94]: Added `LoadOptions`. Here is my IDE suggesting the `All` member. I added this one explicitly as it's a known property but can be omitted if preferred. 

image

**Review by juliofarah — APPROVED** [review]

**Pull request merged**